cypress/support/step_definitions/testSetup.md

```diff
@@ -1,15 +1,60 @@
 # TestSetup Glue
-## I initialize a {string} test
 
-### Purpose: initalize a test during prerequisite
+## the environment has been set up for {string} tests
+
+### Purpose: Sets up the environment for the specified test based on the test name.
+
+#### Below are the tasks that are performed based on the test name:
+
+- This glue will used to initialize the firebolt object for the dynamic calls.
```

Comment:
instead of explaining here let's just link to the section in dynamicObjects.md that does is explaining the same thing. line 11-41

Reply:
Also we say it will be used to initialize dynamic objects but it may not right? We can use this glue code to potentially set those objects. However if we cannot find one no issues occur.

Reply:
updated